### PR TITLE
Project2 ready for review

### PR DESCRIPTION
**Pull request by selfbd**
Completed Project 2. Ready for code review.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -66,9 +65,39 @@ void variableDeclarations()
...
+    ignoreUnused(numberTwo);
+    ignoreUnused(numApples);
+    ignoreUnused(numCars);
+    ignoreUnused(numKids);
+    ignoreUnused(pi);
+    ignoreUnused(temperature);
+    ignoreUnused(marathonDistanceMiles);
+    ignoreUnused(bigPi);
+    ignoreUnused(voltageV);
+    ignoreUnused(pressureT);
```
Comment (matkatmusic) [inline main.cpp]: - [ ] pass them all to a single call of `ignoreUnused`

File: main.cpp [hunk 2]
```diff
@@ -83,70 +112,132 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    chill(1440);
     
     ignoreUnused(carRented);
+    ignoreUnused(roomTempC);
+    ignoreUnused(roomIsCool);
+    ignoreUnused(tempC);
+    ignoreUnused(averageScore);
+    ignoreUnused(splDbAFast);
+    ignoreUnused(rh);
+    ignoreUnused(letterGrade);
```
Comment (matkatmusic) [inline main.cpp]: - [ ] pass them all to a single call of `ignoreUnused`

File: main.cpp (hunk 1 above)
Comment (selfbd) [reply main.cpp]: Requested changes complete

File: main.cpp (hunk 2 above)
Comment (selfbd) [reply main.cpp]: Requested changes complete

File: main.cpp [hunk 3]
```diff
@@ -83,70 +98,135 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
     
     ignoreUnused(carRented);
+
+    ignoreUnused(roomTempC);
+    ignoreUnused(roomIsCool);
+    ignoreUnused(tempC);
+    ignoreUnused(averageScore);
+    ignoreUnused(splDbAFast);
+    ignoreUnused(rh);
+    ignoreUnused(letterGrade);
```
Comment (matkatmusic) [inline main.cpp]: - [ ] You need to remove these calls after you added line 228

Comment (selfbd) [reply main.cpp]: Single calls to ignoreUnused() removed in main()

**Pull request merged**